### PR TITLE
feat: Introduce activationThreshold/minMetricValue for Datadog Scaler

### LINKED ISSUE #3232: Migrate e2e test for Datadog to Go
### Proposal

Relates to https://github.com/kedacore/keda/issues/2357, https://github.com/kedacore/keda/issues/2737.

### Use-Case

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge_turrado@hotmail.es>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added

Fixes #3232 

Related to #2800 
Related to https://github.com/kedacore/keda-docs/pull/812

### CODE REVIEW

File: pkg/scalers/datadog_scaler.go [hunk 1]
```diff
@@ -108,6 +109,15 @@ func parseDatadogMetadata(config *ScalerConfig) (*datadogMetadata, error) {
 		return nil, fmt.Errorf("no queryValue given")
 	}
 
+	meta.activationQueryValue = 0
```
[the remark this answers is not in the document]
Comment (JorTurFer) [reply pkg/scalers/datadog_scaler.go:112]: sure! I'll add to every PR that is already opened for this and also in another PR to other scalers

**Review by zroubalik — APPROVED** [review]

**Pull request merged**